CESM.cfg

```diff
@@ -6,9 +6,10 @@ required = True
 
 [cism]
 local_path = components/cism
-protocol = svn
-repo_url = https://svn-ccsm-models.cgd.ucar.edu/glc
-tag = trunk_tags/cism2_1_40
+protocol = git
+repo_url = https://github.com/ESCOMP/cism-wrapper
+branch = add_manage_externals_file
+externals = CISM.cfg
```

Comment:
@bandre-ucar do I need to explicitly point to an `externals` file like this, or can it be auto-detected if I name it appropriately?